spec/javascripts/helpers-sidebar_spec.js

```diff
@@ -864,6 +833,18 @@ describe('Helpers.Blocks.js', function () {
             expect(instance.setVal).toHaveBeenCalledWith('.fx-location-type', 'test-location')
           })
 
+          it('travel expense type: should define distanceLookupEnabled on change', function () {
+            const selector = '.fx-travel-expense-type select'
+            spyOn(instance, 'statemanager')
+            expect(instance.distanceLookupEnabled).not.toBeDefined()
+
+            instance.bindListners()
+            $(selector).trigger('change')
+
+            expect(instance.distanceLookupEnabled).toEqual(false)
+            expect(instance.statemanager).toHaveBeenCalled()
+          })
+
```

Comment:
Does this test replace all five of the tests that you've removed? If not is it possible to add similar new tests so that we keep up the same level of coverage?

Reply:
I believe existing unit tests already test the event handlers by checking that the callback does as expected. Therefore not all 5 failing tests need to be rewritten I think (some are duplicating tests)